build.gradle

```diff
@@ -1,29 +1,36 @@
-apply plugin: 'java'
-apply plugin: 'eclipse'
+plugins {
+	id 'java-library'
+	id 'eclipse'
+	id 'idea'

```

Comment:
Since you added IDEA here, could you please adjust README.md to not only mention Eclipse anymore? This would be helpful for all IDEA users.

Reply:
Good point